### PR TITLE
vjs 7 release post

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Comment by gkatsev** [comment]
Deploy preview for *videojs-blog* ready!

Built with commit ec1efd101ab6eebdfd20d710731f4c3dae0a68fa

https://deploy-preview-29--videojs-blog.netlify.com

**Comment by gkatsev** [comment]
Preview link: https://deploy-preview-29--videojs-blog.netlify.com/video-js-7-is-here/

File: source/_drafts/Video-js-7-is-here.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+https://unpkg.com/video.js@7.0.0/dist/video-js.css
+```
+```
+https://cdnjs.cloudflare.com/ajax/libs/video.js/7.0.0/video.min.js
+https://cdnjs.cloudflare.com/ajax/libs/video.js/7.0.0/video-js.css
+```
+
+## VHS and Video.js
+Video.js will be bundling VHS by default for ease of use for new users. However, some people don't want VHS or are using another plugin. For this, we have a separate build of Video.js which doesn't include VHS.
+You can get it from our CDN or another CDN
```
Comment (misteroneill) [inline source/_drafts/Video-js-7-is-here.md]: Minor detail, but paragraphs really ought to have two newlines between them, so they result in actual `<p>`s instead of `<br>`s. Helps with readability.

File: source/_drafts/Video-js-7-is-here.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+https://vjs.zencdn.net/7.0.0/alt/video.core.min.js
+```
+```
+https://unpkg.com/video.js@7.0.0/dist/alt/video.core.min.js
+```
+```
+https://cdnjs.cloudflare.com/ajax/libs/video.js/7.0.0/alt/video.core.min.js
+```
+
+Finally, expect a more details post about VHS in the weeks to come.
```
Comment (misteroneill) [inline source/_drafts/Video-js-7-is-here.md]: Should this be `more details *in a* post`?

File: source/_drafts/Video-js-7-is-here.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## What the future holds
+Video.js hasn't really had a roadmap for a long time. With this release, we think it's time to take a step back and create a roadmap.
+The roadmap will take into account currently open issues and internal and external feature requests. In addition, we're going to look at currently available features and see if it's worth refactoring them with the more modern web in-mind.
+Later down the line, we'd also want to make a roadmap for a Video.js 8. Given that this release only adds VHS and removes old browsers, there's plenty of already deprecated features that it's time to remove.
+
+## Updates from the [Video.js 7 roadmap][roadmap]
+### Video.js 5.x
+
+As of this release, Video.js 5.x is now officially deprecated. This means that we will no longer work on it or back-port fixes to it but it's still available for usage if you're still using it. We do urge you to update to the latest 5.x if you're on it, or better yet, upgrade to 6.x or 7 if you can.
```
Comment (misteroneill) [inline source/_drafts/Video-js-7-is-here.md]: `still available for usage if you're still using it` feels redundant. Maybe just `still available for usage`?

Also, `latest 5.x if you're on it, or better yet, upgrade to 6.x or 7 if you can` seems like it may be too wordy - maybe `latest 5.x, or better yet, upgrade to 6.x or 7`?

File: source/_drafts/Video-js-7-is-here.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+### Build Tools
+In the [roadmap][] post, we talked about potentially switching to webpack from rollup for better support of importing VHS into Video.js. However, while doing [this work](https://github.com/videojs/video.js/pull/5033), we found out that the filesize of Video.js will increase by about 20%, which isn't a trivial amount, not to mention that the filesize is already going to increase from the default inclusion of VHS.
```
Comment (misteroneill) [inline source/_drafts/Video-js-7-is-here.md]: Missing roadmap link? Also, we should probably capitalize Rollup since it's capitalized on their website. Webpack is probably fine as-is since that's their "branding".

File: source/_drafts/Video-js-7-is-here.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Eventually, after much work, we were able to figure out how to bundle VHS using rollup as well and we are continuing to use rollup for our builds.
+
+### Google Analytics
+Video.js 7's CDN builds will no longer send any data to Google Analytics via our stripped down pixel tracking.
+Video.js 6 and below CDN builds will continue sending data but versions 6.8 and above will honor the Do Not Track option that users can set in their headers before sending the data.
```
Comment (misteroneill) [inline source/_drafts/Video-js-7-is-here.md]: Maybe re-iterate that users can opt-out in 6:

`will continue sending data but versions 6.8` to `will continue sending data unless you opt-out, but versions 6.8`

**Review by misteroneill — COMMENTED** [review]
A couple thoughts.

File: source/_drafts/Video-js-7-is-here.md (hunk 2 above)
Comment (kcj382) [reply source/_drafts/Video-js-7-is-here.md]: ... or, "... a more detailed post ..." ?

File: source/_drafts/Video-js-7-is-here.md (hunk 3 above)
Comment (kcj382) [reply source/_drafts/Video-js-7-is-here.md]: perhaps "there's plenty" should be "there're plenty" ?

File: source/_drafts/Video-js-7-is-here.md (hunk 2 above)
Comment (gkatsev) [reply source/_drafts/Video-js-7-is-here.md]: Yeah, this was supposed to be "a more detailed post"

File: source/_drafts/Video-js-7-is-here.md (hunk 4 above)
Comment (gkatsev) [reply source/_drafts/Video-js-7-is-here.md]: it's a shorthand for the bottom links thing, since the reference will is also `roadmap`, you don't need to double it.
